From 2cd2b8fe43036a97f1051c5aa76fd5ed28e7ed6c Mon Sep 17 00:00:00 2001
From: Rob Crittenden <rcritten@redhat.com>
Date: Wed, 16 Oct 2024 15:39:31 -0400
Subject: [PATCH] Set required version of 389-ds for VLV fix on F40/41

Require builds that contain the fixes for VLV handling in
https://bugzilla.redhat.com/show_bug.cgi?id=2317851

Signed-off-by: Rob Crittenden <rcritten@redhat.com>
Reviewed-By: Florence Blanc-Renaud <flo@redhat.com>
Reviewed-By: Alexander Bokovoy <abokovoy@redhat.com>
Reviewed-By: Rafael Guterres Jeffman <rjeffman@redhat.com>
---
 freeipa.spec.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/freeipa.spec.in b/freeipa.spec.in
index 72d7013a6c49873f4a59734c684c6c5510e669d0..1283ba847f60cd335851d7e846536a4a6c14b071 100755
--- a/freeipa.spec.in
+++ b/freeipa.spec.in
@@ -127,6 +127,10 @@
 # version supporting LMDB and lib389.cli_ctl.dblib.run_dbscan utility
 %if 0%{?fedora} < 34
 %global ds_version 1.4.4.16-1
+%elif 0%{?fedora} == 40
+%global ds_version 3.0.4-3
+%elif 0%{?fedora} >= 41
+%global ds_version 3.1.1-3
 %else
 %global ds_version 2.1.0
 %endif
-- 
2.47.0

